vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.h

```diff
@@ -103,7 +103,7 @@ class ReactInstanceWin final : public Mso::ActiveObject<IReactInstanceInternal,
       const std::shared_ptr<winrt::Microsoft::ReactNative::NativeModulesProvider> &nativeModulesProvider,
       const std::shared_ptr<winrt::Microsoft::ReactNative::TurboModulesProvider> &turboModulesProvider) noexcept;
   void Initialize() noexcept override;
-  ~ReactInstanceWin() override;
```

Comment:
>override [](start = 22, length = 8)

Please do not remove. The 'override' keyword must be here because the base class already defines the virtual destructor and we better be explicit about it.